checklist.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+permalink: /checklists/training/
+title: Checklist
+---
+## Scheduling the Event
+
+1.  Decide if it will be in person, online for one site, or online for several.
+1.  Talk through expectations with the host(s).
+    *   If it is in person, make sure the host knows they're covering travel costs.
+    *   Determine who is allowed to attend.
```

Comment:
Depending on the context of the training you may need to help the host develop a list of priorities for who is allowed to attend. For example: the trainee instructors should have had some experience at workshops, either as participants or helpers, or even second/third instructors.